### PR TITLE
Adapt some tests scripts from lxc-ci to Openstack

### PR DESCRIPTION
**Pull request by simondeziel**
From the `main.sh` test list, `cgroup1` is failing on `jammy`. Previously in `lxc-ci` we used to test `cgroup2` along with `jammy` but I think that's already `jammy`'s default which is why I switched to `cgroup1`:

```
./bin/openstack-run jammy cgroup1 tests/cgroup
```

However, the following works:

```
./bin/openstack-run focal default tests/cgroup
```

So it seems like a regression to me.

### CODE REVIEW

**Review by tomponline — CHANGES_REQUESTED** [review]
Looks like this duplicates some of the tests I cleaned up (partially} today and pushed in as part of the lxd 5.16 manual test run. Please can you update it so there's n9 duplication. Thanks

File: cloud-init.user-data.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#cloud-config
+apt:
+  # no need for backports/security nor restricted/multiverse
```
Comment (tomponline) [inline cloud-init.user-data.yaml:3]: Could you explain why in the comments as well as what.

**Review by tomponline — COMMENTED** [review]
Looking good! Thanks

**Comment by simondeziel** [comment]
I couldn't find the duplicated manual tests you referred to, would you mind pointing them out, please?

**Comment by tomponline** [comment]
> 

https://github.com/canonical/lxd-ci/blob/main/tests/test-lxd-network-bridge-firewall
https://github.com/canonical/lxd-ci/blob/main/tests/test-lxd-network-ovn
https://github.com/canonical/lxd-ci/blob/main/tests/test-lxd-storage-buckets

I missed interception actually so would be good to bring that one across.

**Review by tomponline — APPROVED** [review]

**Pull request merged**